### ISSUE TITLE
%get-current-time dispatch-macro-character error when loading with lispworks 7.0/7.1

### ISSUE THREAD

**Issue by paulvaneecke** [dlowe-net/local-time#74]
When loading the local-time system in Lispworks 7.0 and 7.1 (on OS X): the following error occurs:

`++++ Error between functions:
  Subcharacter #\> not defined for dispatch char #\#.`

The problem is in the file `local-time.lisp`, in the function `%get-current-time dispatch-macro-character`,  in the following block of code.

```
  #+(and ccl windows)
  (let ((unix-epoch-filetime 116444736000000000))
    (ccl:rlet ((time #>FILETIME))
      (#_GetSystemTimeAsFileTime time)
      (let* ((filetime
               (logior (ccl:pref time #>FILETIME.dwLowDateTime)
                       (ash (ccl:pref time #>FILETIME.dwHighDateTime) 32)))
             (filetime (- filetime unix-epoch-filetime)))
        (multiple-value-bind (secs 100ns-periods)
            (floor filetime #.(round 1e7))
          (values secs (* 100ns-periods 100))))))
```

Although this block of code is not even evaluated in lispworks, the error still occurs when reading.

Any ideas on how to solve this? I am new to the dispatching of macro characters.

(On the same system: no problems when using ccl)

**Comment by dlowe-net** [comment]
This was fixed as a side effect of PR #75. You shouldn't have this trouble anymore.

**Comment by paulvaneecke** [comment]
This solves the problem indeed, thanks a lot!
